modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/resources/META-INF/resources/manage_user_associated_data_summary.jsp

```diff
@@ -47,7 +47,7 @@ int step = (int)request.getAttribute(UserAssociatedDataWebKeys.MANAGE_USER_ASSOC
 
 					<aui:button
 						disabled="<%= step != 1 %>"
-						onClick='<%= renderResponse.getNamespace() + "confirmAction('" + deactivateUserURL.toString() + "', 'are-you-sure-you-want-to-deactivate-the-user')" %>'
+						onClick='<%= renderResponse.getNamespace() + "confirmAction('" + deactivateUserURL.toString() + "', '" + UnicodeLanguageUtil.get(request, "are-you-sure-you-want-to-deactivate-the-user") + "')" %>'
```

Comment:
I'm not familiar with `UnicodeLanguageUtil`.  We usually use `LanguageUtil`.  Was there a specific reason you chose to use it?